data/risingwave/models/30_gold/energy/electrical_soc_hourly.sql

```diff
@@ -1,4 +1,22 @@
 {{ config(materialized='materialized_view') }}
 
--- This temporarily uses dummy data for testing and visualization purposes. This model will be updated to process data from the silver layer.
- SELECT * FROM {{ ref('electrical_soc_dummy_data')}}
+-- Calculate stored energy (Wh) by calculating average values per hour per topic.
+SELECT
+    soc_data.electrical_system AS electrical_system,
+    soc_data.group_name AS group_name,
+    soc_data.sub_group_name AS sub_group_name,
+    soc_data.topic AS topic,
+    window_start AS hour,
+    AVG(soc_data.stored_energy) AS soc
+FROM
+TUMBLE (
```

Comment:
```suggestion
TUMBLE(
```

TUMBLE is just a function